stdlib/ballerina-config/pom.xml

```diff
@@ -32,6 +32,13 @@
     <name>Ballerina - Config API</name>
     <url>http://ballerinalang.org</url>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.ballerinalang</groupId>
+            <artifactId>ballerina-core</artifactId>
```

Comment:
What's the need to add the core as a dependency? The config module was intentionally kept free of dependencies to other modules to avoid cyclic dependencies

Reply:
Ok. Got it. I wanted to create a new Ballerina Exception -> BallerinaSecurityException so it was needed. It seems then we have to create it in the core by throwing the original exceptions from the config API.